apps/meteor/ee/app/livechat-enterprise/server/hooks/applyDepartmentRestrictions.ts

```diff
@@ -25,7 +27,7 @@ callbacks.add(
 		}
 
 		cbLogger.debug('Applying department query restrictions');
-		return addQueryRestrictionsToDepartmentsModel(originalQuery);
+		return addQueryRestrictionsToDepartmentsModel(originalQuery, userId);
```

Comment:
should a `userId` be provided on the line below as well?

https://github.com/RocketChat/Rocket.Chat/blob/32c6b0b308336f6db68aad35c16c5c2224865807/apps/meteor/app/livechat/server/lib/Livechat.js#L288

I mean, even though `userId` is optional in this callback, it doesn't do anything if it is not provided..